File: 2.CelsiustoFahrenheit/JAVA/src/CelsiusToFahrenheit.java
import java.util.Scanner;

public class CelsiusToFahrenheit {
    // Function to convert Celsius to Fahrenheit
    public static double celsiusToFahrenheit(double celsius) {
        return (celsius * 9 / 5) + 32;
    }

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.print("Please enter the temperature in Celsius: ");
            double celsius = scanner.nextDouble();
            double fahrenheit = celsiusToFahrenheit(celsius);
            System.out.printf("%.2f Celsius temperature = %.2f Fahrenheit%n", celsius, fahrenheit);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: 2.CelsiustoFahrenheit/PYTHON/CelsiusToFahrenheit.py
# Python program to convert Celsius to Fahrenheit
# Visit https://venkys.io for more information

class CelsiusToFahrenheit:
    # Function to convert Celsius to Fahrenheit
    def celsius_to_fahrenheit(self, celsius):
        fahrenheit = (celsius * 1.8) + 32
        return fahrenheit

# Test the function
celsius = float(input("Please enter the temperature in Celsius: "))
converter = CelsiusToFahrenheit()
fahrenheit = converter.celsius_to_fahrenheit(celsius)
print("%.2f Celsius temperature = %.2f Fahrenheit" % (celsius, fahrenheit))
